src/theme/variables.scss

```diff
@@ -475,15 +451,4 @@ $horizontal: 90deg;
     var(--combo-box-gradient-passive-start) 34.6%,
     var(--combo-box-gradient-passive-end) 105.68%
   );
-
-  --ion-color-help-tooltip: #{color.adjust($primary, $hue: -8%, $saturation: 12%, $lightness: 9%)};
-  --ion-color-slider: #{color.adjust($primary, $hue: -8%, $saturation: 12%, $lightness: 9%)};
-  --ion-btn-dark-box-shadow: 0px 4px 4px #00000040;
```

Comment:
this is actually identical to `--ion-default-box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25)` – one is using a hexadecimal value to define the alpha property and the other is using a proportion of 1. So have deprecated `--ion-btn-dark-box-shadow`

Reply:
sounds good, yeah given the choice I much prefer use of rgba or hsla colours where possible, nice to be able to do colour operations on and have a reasonable expectation of what will happpen